react/components/ShippingSimulator/components/ShippingTableRow.js

```diff
@@ -4,18 +4,18 @@ import { intlShape, injectIntl } from 'react-intl'
 import TranslateEstimate from 'vtex.shipping-estimate-translator/TranslateEstimate'
 import classNames from 'classnames'
 
-import styles from '../styles.css'
+import shippingSimulator from '../shippingSimulator.css'
 
 const ShippingTableRow = ({ name, shippingEstimate, price, intl }) => {
   const etaClassName = classNames(
-    `${styles.shippingTableCell} pv1 ph3 t-small c-muted-2`,
+    `${shippingSimulator.shippingTableCell} pv1 ph3 t-small c-muted-2`,
     {
       tc: shippingEstimate === undefined,
     }
   )
 
   const valueClassName = classNames(
-    `${styles.shippingTableCell} pv1 ph3 t-small c-muted-2`,
+    `${shippingSimulator.shippingTableCell} pv1 ph3 t-small c-muted-2`,
     {
       tc: price === undefined,
```

Comment:
I know that isnt your task, but its a small alteration.
If goal is know if a variable is `undefined` the correct way is more like `typeof foo === "undefined"`. This little mistake change the semantic of the command and can introduce unexpected behavoir. Can you check if this make sense in our case?

Reply:
It makes senses, just changed it!